charts/dvpe-deployment-gloo/CHANGELOG.md

```diff
@@ -7,7 +7,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [2.0.0] - 2021-??-??
+## [2.1.0] - 2021-04-14
+
+### Added
+
+* Enable List and String for VirtualService Domain.
+* Helper Function to create authorizationCode.appUrl from the given Domain.
+
```

Comment:
as discussed before, I would mention here that the changes in `charts/dvpe-deployment-gloo/templates/_helpers.tpl` are deprecated and will be removed with the next major release of the chart